src/index.js

```diff
@@ -132,7 +132,7 @@ const microdown = function () {
       (match, k, text) => tag([, 'u', 's', 'del'][k.length], inline(text)),
 
       // replace remaining newlines with a <br>
-      /  \n|\n  /g,
+      /  \n|\n\n|\n  /g,
```

Comment:
an empty line shouldn't be a <br> but rather it seperates paragraphs, like it does currently.